yang/frr-ripd.yang

```diff
@@ -298,83 +298,83 @@ module frr-ripd {
           '(./receive = "2" and ./send = "2") or ' +
           '(./receive = "1-2" and ./send = "2")';
       }
-    }
-
-    /*
-     * Operational data.
-     */
-    container state {
-      config false;
-      description
-        "Operational data.";
 
-      container neighbors {
+      /*
+       * Operational data.
+       */
+      container state {
+        config false;
         description
-          "Neighbor information.";
-        list neighbor {
-          key "address";
+          "Operational data.";
+
```

Comment:
should not the yang/ietf/frr-deviations-ietf-rip.yang file be upgraded, in regards to the yang change ?

Reply:
same question for yang/ietf/frr-deviations-ietf-interface.yang

Reply:
Good catch, `frr-deviations-ietf-rip.yang` and `frr-ietf-translator.json` need to be updated. Will do that now.